core/deliverservice/config.go

```diff
@@ -124,6 +130,24 @@ func (c *DeliverServiceConfig) loadDeliverServiceConfig() {
 	}
 	c.BlockGossipEnabled = enabledConfigOptionMissing || viper.GetBool(enabledKey)
 
+	blockCensorshipTimeout := "peer.deliveryclient.blockCensorshipTimeout"
+	blockCensorshipTimeoutOptionMissing := !viper.IsSet(blockCensorshipTimeout)
+	if blockCensorshipTimeoutOptionMissing {
```

Comment:
is there a unit test that covers the case where the new config keys are missing, and check that the default is applied?